### PR TITLE
update(SendFilesLayout): Fix send files layout UI from chats

### PR DESCRIPTION
**Pull request by lgmarchi**
### 1. Fix UI on Send Files Layout 
 - Before: 
 
image

 - After: 
image

### Which issue(s) this PR fixes 🔨

- Resolve #

### CODE REVIEW

File: ui/src/layouts/style.scss [hunk 1]
```diff
@@ -146,75 +152,75 @@
...
+  display: inline-flex;
+  flex-direction: column;
+  overflow-y: scroll;
+  padding-right: var(--gap);
+  overflow-x: hidden;
+}
+
+.files-breadcrumbs {
+  display: inline-flex;
+  margin: var(--gap) 0;
```
Comment (InfamousVague) [inline ui/src/layouts/style.scss]: We should remove this one

**Review by InfamousVague — COMMENTED** [review]
One small change, looks good!

File: ui/src/layouts/style.scss (hunk 1 above)
Comment (lgmarchi) [reply ui/src/layouts/style.scss]: What exactly to remove @InfamousVague ?

**Review by phillsatellite — APPROVED** [review]

File: ui/src/layouts/style.scss (hunk 1 above)
Comment (lgmarchi) [reply ui/src/layouts/style.scss]: Fixed my friend!

**Pull request merged**